test/box-luatest/gh_6260_add_builtin_events_test.lua

```diff
@@ -13,7 +13,7 @@ g.test_subscriptions_outside_box_cfg = function()
         local result_no = 0
         local watcher = box.watch(val,
                                   function(name, state)
-                                      assert(name == val)
+                                      t.assert(name == val)
```

Comment:
Why not `assert_equals()`?

Reply:
That's a typo, thanks for noticing! Fixed.